src/data/events.ts

```diff
@@ -8,27 +8,90 @@ import EventImage1 from '@assets/events/projects-event-image-1.jpeg';
 import EventImage2 from '@assets/events/projects-event-image-2.jpeg';
 import EventImage3 from '@assets/events/projects-event-image-3.jpeg';
 import type { GetImageResult } from 'astro';
-import * as qs from 'qs';
-import type { Event as EventCMS } from './payload-types.ts';
+import qs from 'qs';
+import type { Address, Event as EventCMS, Media } from './payload-types.ts';
 
-export interface Event {
+export interface EventCalendarEntry {
   title: string;
   date: Date;
   description: string;
-  content_html?: string;
   link?: string;
-  slug?: string;
-  image: {
-    src: Awaited<ReturnType<typeof getImage>>;
-  };
+  image: YVImage;
+}
+
+export interface EventPage {
+  title: string;
+  start: Date;
+  end: Date;
+  slug: string;
+  content_html: string;
+  heroImage?: YVImage;
+  address: Address;
+  audience: string;
+  cost: string;
+  team: { name: string; job: string; bio: string; image: YVImage }[];
+  registrationLink: string;
+  timetable?: YVImage;
 }
 
-export async function getNext3Events(): Promise<[Event, Event, Event]> {
-  const image1 = await getImage({ src: thirdEventImage });
-  const image2 = await getImage({ src: calendarCoverImage });
-  const image3 = await getImage({ src: pastEvent });
+export type YVImage = {
```

Comment:
Not sure if YVImage is a fitting name for that type, but I couldn't think of a better one.

Reply:
Could also just be `ImageWithAltText` or something